modules/publicize/publicize.php

```diff
@@ -565,4 +565,33 @@ function test_publicize_conns() {
 
 		wp_send_json_success( $test_results );
 	}
+
+	protected static function build_sprintf( $args ) {
+		$search = array();
+		$replace = array();
+		foreach ( $args as $k => $arg ) {
+			if ( 0 == $k ) {
+				$string = $arg;
+				continue;
+			}
+			$search[] = "%$arg%";
+			$replace[] = "%$k\$s";
+		}
+		return str_replace( $search, $replace, $string );
+	}
+}
+
+function publicize_calypso_url() {
```

Comment:
This was added in r128505-wpcom, but is not used anywhere in Jetpack.  Bringing over here just for consistency in files.